src/components/Locations.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import "../App.css";
+
+
+class Locations extends React.Component {
+    constructor() {
+        super()
+
+        this.state = {
+            location: []
+        }
```

Comment:
```suggestion
        this.state = {
            locations: [], //<-- locations should be plural. It's an array of objects.
            showLocations: false
        }
```
I want to point out a couple things about your `this.state`:
1) location**s** (plural!!) is going to be an array of **multiple** location objects.
2) you need a way to keep track of whether your locations will **show** on the page or if they'll be **hidden**. `showLocations` is a piece of state, represented as a boolean, that will determine whether or not to show the list of locations.